src/SprykerEco/Yves/AmazonPay/Plugin/Provider/AmazonPayControllerProvider.php

```diff
@@ -58,6 +60,7 @@ protected function defineControllers(Application $app)
         $this->createController('/{amazonpay}/get-shipment-methods', static::GET_SHIPMENT_METHODS, static::BUNDLE_NAME, 'Payment', 'getShipmentMethods')
             ->assert('amazonpay', $allowedLocalesPattern . 'amazonpay|amazonpay')
             ->value('amazonpay', 'amazonpay');
+        $this->createController('/amazonpay/place-order', static::PLACE_ORDER, static::BUNDLE_NAME, 'Payment', 'placeOrder');
```

Comment:
shouldn't this be only post restricted?

Reply:
Also in other routes we have `{}`. Mb we need them here too?

Reply:
As I see {} need for not ajax calls. I can add it but actually, it's not required here.